features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -239,9 +254,8 @@ bool ATHandler::find_urc_handler(const char *prefix)
 
 void ATHandler::event()
 {
-    if (!_oob_queued) {
-        _oob_queued = true;
-        (void) _queue.call(Callback<void(void)>(this, &ATHandler::process_oob));
+    if (_event_id <= 0) {
```

Comment:
I'm a bit nervous that the code uses `-1` as its invalid value, but tests for `<=0` and `>0`. Is `0` valid or not?

Looking at the docs, `0` is documented as invalid, so check against that specifically. I don't see it stated that valid IDs are positive.